quickstarts/microsoft.netapp/anf-sap-hana/anf-sap-hana-storage/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
readme says we need an avset for pinning, but there is no avset in this template

Reply:
avset is required for manual pinning, more details in email thread.